lib/rules/no-curly-component-invocation.js

```diff
@@ -126,6 +128,11 @@ module.exports = class NoCurlyComponentInvocation extends Rule {
             return;
           }
 
+          if (BUILT_INS.includes(name)) {
```

Comment:
`BUILT_IN.includes` is not a function (CI failure), likely because this is a set now

```suggestion
          if (BUILT_INS.has(name)) {
```

Reply:
thought I changed that ... 🤔